web-console/src/views/segments-view/segments-view.tsx

```diff
@@ -345,6 +346,9 @@ export class SegmentsView extends React.PureComponent<SegmentsViewProps, Segment
               orderByClause,
               `LIMIT ${pageSize * 1000}`,
             ]);
+
+            // This is needed because there might be an IN filter with a lot of intervals, the number of which exceeds the default inFunctionThreshold
+            sqlQueryContext.inFunctionThreshold = 1000;
```

Comment:
Will the console never do an `IN` of more than 1000 values when writing `sys` queries? This should be larger than the largest possible `IN` the console will generate.

Reply:
The IN will have up to `pageSize` things in it so I'll set it to `pageSize + 1`. Somewhere else pageSize is limited to 200 but this way is more robust